### ISSUE TITLE
unchecking "small find" should actually delete the small find record

### ISSUE THREAD

**Issue by urapadmin** [urapadmin/kiosk#1137]
otherwise the director's view and even filemaker recording would still think there is a small find on occasion.

**Comment by urapadmin** [comment]
### @test recording 10.7
- [ ] done. can be tested best by checking a small find, adding data in that tab, unchecking it again and then checking it again and the "small find" form should be empty.

**Comment by lbestock** [comment]
When I uncheck the "small find" it indeed is no longer marked as small find, and I cannot enter the small find form. But when I re-check the small find button it a) does not ask me again if I am sure if I want to make it a small find, as it would if the button had not previously been clicked, and then b) when I go into the small finds tab the description I had previously entered is present, which is not the behavior you wanted to see.

A side check worked: a previously small found CM that is then un-small finded has the proper status in the director's view after synchronization.

**Comment by urapadmin** [comment]
no, sounds as if this is not working.

**Comment by urapadmin** [comment]
I really cannot reproduce this problem. I logged in as a standard user and all and still can uncheck the box and the record gets deleted. I do get the question again if I want to make this a small find and the small find data is empty.
I am on a PC and on 10.8 but at least the latter should not make a difference.
Are you sure you are in the right version? Does it work on your mac? Perhaps it is an ipad issue (though, I don't see how).

**Comment by lbestock** [comment]
I was testing in PVD on the iPad but it behaves differently now when I try it again on a new download. Previously I had the small finds tab even when I couldn't enter it; now I only have the tab at all if the button is checked. And now I get all the proper warnings and the right behavior. Hm. I wish I had recorded what version I was in. I wish I did not hit so many hiccups in PVD (this download I am not getting the problem with invisible photos as I had with the last download).
